library/ord-tree.lisp

```diff
@@ -436,7 +450,9 @@ The result tree may be in an intermediate state with a double-black node."
        (cell:push! stack (Expand more))
        (cell:push! stack (Yield elt))
        (cell:push! stack (Expand less))
-       Unit)))
+       Unit)
+      (_
```

Comment:
Back to the first one; this is outside of `remove`, so `DoubleBlackEmpty` should never be present.